_posts/ianmoore/2016-04-27-ian-open-source.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+author: touchwick
+title: "Ian's Open Source Contribution"
+---
+For my open source contribution, I decided to contribute to the Fedora project. Like Ga Kai, I heard it mentioned at a meetup, which piqued my interest. I was a Fedora user several years ago, just before version 17, Beefy Miracle. It was fun! I had a great time learning about Linux with it. But it didn't quite have everything I needed, so I abandoned it for my Mac/Chromebook combo. Anyway, it was fun to check back in and see how things have changed.
+
+I used the Fedora tagger (which, incidentally, I do have a screenshot of. I do not know how to add screenshots to these posts. Tips would be appreciated.) and went through around 100 packages. It was actually a ton of fun. I got to poke around through some packages and see what was or wasn't there while deciding if the tags attached to them applied. The whole thing is kind of like a game. Not that it matters or anything, but my rank is still 275, and I would like to get it higher. This was fun!
```

Comment:
I use imgr for easy image hosting but you can put the image anywhere.  Once the image is on the internet the markdown syntax to embed is `![description](url/to/image/file)`  Make sure to link to the actual file, not a webpage.